clusterloader2/testing/experiments/ignore_known_gce_container_restarts.yaml

```diff
@@ -7,3 +7,7 @@ RESTART_COUNT_THRESHOLD_OVERRIDES: |
   kube-proxy: 2
   metadata-proxy: 2
   prometheus-to-sd-exporter: 2
+
+  # CoreDNS runs multiple copies. We might be unlucky with hardware here too
+  # Example: https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-scale-performance/1190313018366365696
+  coredns: 2
```

Comment:
Shouldn't we set it to 1 in such case?

@pierewoj, why do we use 2 in the pods above (kube-proxy and others)?

Reply:
No strong preferences on an actual value, except it shouldn't be 0. We can always tweak it if needed. 

To me 2 for components running on each node sounds like a reasonable value.

Reply:
Main purpose of this check was AFAIK detection crashlooping pods, we extended it a bit to be notified if master pods are restarted even once. I think we might be better off staying on the less aggressive side with the thresholds for now to reduce number of false positives.

See http://perf-dash.k8s.io/#/?jobname=gce-5000Nodes&metriccategoryname=SystemPodMetrics&metricname=Load_SystemPodMetrics&RestartCount=RestartCount

Reply:
Thanks, that makes sense. I agree that most likely the cost of chasing each 2 restarts it's not worth the potential gain. 

Can we explain this (why we use 2) in this file for posterity?

Reply:
Yes